### PR TITLE
chore(sdk): fix 1.8.17 release notes

### PR DESCRIPTION
**Pull request by connor-mccarthy**
**Description of your changes:**
Moves an item from "Breaking Changes" to "Bug Fixes and Other Changes".

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: sdk/RELEASE.md [hunk 1]
```diff
@@ -14,11 +14,10 @@
 ## Documentation Updates
 # 1.8.17
 ## Major Features and Improvements
-
+* Allow ephemeral storage to be set via component output parameter [#8428](https://github.com/kubeflow/pipelines/pull/8428)
```
Comment (chensun) [inline sdk/RELEASE.md:17]: Probably not a major feature or improvement. I would probably categorize it as other changes. But I don't have a strong opinion here.

**Review by chensun — COMMENTED** [review]
/lgtm

File: sdk/RELEASE.md (hunk 1 above)
Comment (connor-mccarthy) [reply sdk/RELEASE.md]: Agreed. Done.

**Review by chensun — APPROVED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**